cluster/distributedtask/manager.go

```diff
@@ -0,0 +1,229 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package distributedtask
+
+import (
+	"context"
+	"encoding/json"
+	"sync"
+	"time"
+
+	"github.com/jonboulle/clockwork"
+	"github.com/pkg/errors"
```

Comment:
Can we instead of this package use instead `fmt.Errorf(...)`? This is a very old (and already archived) project

Reply:
sure